src/openslide-vendor-dicom.c

```diff
@@ -711,6 +726,76 @@ static void add_properties(openslide_t *osr, struct dicom_level *level0) {
   }
 }
 
+static bool quickhash_level_pixels(struct _openslide_hash *hash,
+                                   struct dicom_level *level) {
+  double d;
+  if (!get_tag_decimal_str(level->file->metadata, NumberOfFrames, 0, &d)) {
```

Comment:
Isn't this an Integer String?

Reply:
Yes, though a double will hold the full range and it saved writing `get_tag_integer_str()`.

Reply:
I added `get_tag_int_str()`.